File: server/models/product.js
'use strict';
module.exports = (sequelize, DataTypes) => {
  const product = sequelize.define('product', {
    name: DataTypes.STRING,
    desc: DataTypes.TEXT,
    method: DataTypes.STRING,
    price: DataTypes.INTEGER,
    duration: DataTypes.STRING,
    image: DataTypes.STRING
  }, {});
  product.associate = function(models) {
    // associations can be defined here
    product.belongsTo(models.users);
    product.hasOne(models.section);
    product.hasMany(models.order);
  };
  return product;
};

File: pinjemin/lib/providers/product.dart
import 'package:flutter/foundation.dart';

class Product with ChangeNotifier {
  final int id;
  final String name;
  final String desc;
  final String method;
  final int price;
  final String duration;
  final String image;
  final DateTime startDate;
  final DateTime endDate;
  final int type;
  

  Product({
    this.id,
    this.name, 
    this.desc,
    this.method,
    this.price ,
    this.duration,
    this.image,
    this.startDate,
    this.endDate,
    this.type
  });
}